src/containers/ContactReady/index.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+
+import LineChart from "../../components/LineChart";
+import { CommonFilters } from "../../components/Chart";
+import MetricsTemplate from "../MetricsTemplate";
+import { ObjectNested, Router } from "../../libraries";
+import {
+  mostAndLeast,
+  normalize,
+  getTemporaryDefaultFilters,
+} from "../../modules/Chart";
+import { TEMP_MIN_DATE } from "../../constants/Charts";
+
+const handleData = data => {
```

Comment:
```
const handleData = data => {
  const localData = ObjectNested.get(data, "timeline", {});
  return {
    globalStats: mostAndLeast(localData),
    chart: normalize(localData, "openIssues"),
  };
};
```
the only change.

Reply:
@magsout OK, let me know if this is correct... 😁